pkgs/games/shattered-pixel-dungeon/deps-experienced.json

```diff
@@ -0,0 +1,244 @@
+{
+ "https://oss.sonatype.org/content/repositories/snapshots/com/badlogicgames/gdx-controllers": {
+  "gdx-controllers-core/2.2.3-SNAPSHOT/gdx-controllers-core-2.2.3-20230427.074808-4": {
+   "jar": "P8AKoxjxVAX4AbiZsF8Z3NRqazNYdvapBBpVbhJT+2A=",
+   "pom": "YrAdNWtXy0wlFltJzoTkCTMQNggFCXK/j0YsSASmh1o="
+  },
+  "gdx-controllers-core/2.2.3-SNAPSHOT/maven-metadata": {
+   "xml": "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<metadata modelVersion=\"1.1.0\">\n  <groupId>com.badlogicgames.gdx-controllers</groupId>\n  <artifactId>gdx-controllers-core</artifactId>\n  <version>2.2.3-SNAPSHOT</version>\n  <versioning>\n    <snapshot>\n      <timestamp>20230427.074808</timestamp>\n      <buildNumber>4</buildNumber>\n    </snapshot>\n    <lastUpdated>20230427074808</lastUpdated>\n    <snapshotVersions>\n      <snapshotVersion>\n        <extension>jar</extension>\n        <value>2.2.3-20230427.074808-4</value>\n        <updated>20230427074808</updated>\n      </snapshotVersion>\n      <snapshotVersion>\n        <extension>pom</extension>\n        <value>2.2.3-20230427.074808-4</value>\n        <updated>20230427074808</updated>\n      </snapshotVersion>\n      <snapshotVersion>\n        <classifier>library</classifier>\n        <extension>jar</extension>\n        <value>2.2.3-20230427.074808-4</value>\n        <updated>20230427074808</updated>\n      </snapshotVersion>\n      <snapshotVersion>\n        <classifier>javadoc</classifier>\n        <extension>jar</extension>\n        <value>2.2.3-20230427.074808-4</value>\n        <updated>20230427074808</updated>\n      </snapshotVersion>\n      <snapshotVersion>\n        <classifier>sources</classifier>\n        <extension>jar</extension>\n        <value>2.2.3-20230427.074808-4</value>\n        <updated>20230427074808</updated>\n      </snapshotVersion>\n    </snapshotVersions>\n  </versioning>\n</metadata>\n"
```

Comment:
Is this supposed to look like this?

Reply:
it was, but not anymore, xml files are constructed by nix code now